### ISSUE TITLE
Unknown transaction opened by db navigator 

### ISSUE THREAD

**Issue by vmalyutin** [dbeaver/dbeaver#6618]
#### System information: 
- Operating system (distribution) and version - Windows
- DBeaver version - 6.1.5.201908191349
- Java version - I don't know
- Additional extensions - No

#### Connection specification:
- Database name and version - Postgres 9.4
- Driver name
- Do you use tunnels or proxies (SSH, SOCKS, etc)? - pgBouncer

#### Describe the problem you're observing:
When I try to close a connection opened from db navogator it asks me if I want to commit or rollback a transaction, but I did not open any transaction explicitly. It was just a simple statement like "select count(*) from MyTable". What is this transaction for?

**Comment by vmalyutin** [comment]
When I try to close a connection opened from db navogator it asks me if I want to commit or rollback a transaction, but I did not open any transaction explicitly. It was just a simple statement like "select count(*) from MyTable". What is this transaction for? 
I connect to Postgres 9.4 DBeaver  version is 6.1.5.201908191349. One note. I think the connection was established via pgBouncer.

**Comment by serge-rider** [comment]
Even simple select statement affects transaction (see #1510).

However, we've been asked about that too often. 
I'll fix behavior of transactional statements calculation, it will track only DML queries.

Commit/rollback will be enabled always (in manual txn mode)

**Comment by vmalyutin** [comment]
Thank you.
I am not sure what txn is, perhaps it means transaction.

Please note, there are two transaction modes in MS SQL Server and PostgreSQL - implicit and explicit. Explicit one is up to me. When I write BEGIN TRANSACTION I go into explicit mode. In this mode I have to close transaction manuallly with COMMIT or ROLLBACK. In other hand, implicit mode is up to server. So, when I did not open transaction explicitly, server will do it automaticlly for me. Also, Read Committed is the default isolation level in PostgreSQL. Also, a SELECT acquire only shared lock which is released right after the statement is complete, regardless the current transaction status. So, under read commited isolation level with implicit transaction mode a completed SELECT do not block anyone. 
In a nutshell. I did not begin transaction, I did a simple select - there mustn't be any open transaction. Only if db navigator began it and did not close on time.

**Comment by kseniiaguzeeva** [comment]
Verified
